code/modules/clothing/suits/armor/bulletproof.dm

```diff
@@ -15,6 +15,27 @@
 	siemens_coefficient = 0.7
 	starting_accessories = list(/obj/item/clothing/accessory/armguards/ballistic, /obj/item/clothing/accessory/legguards/ballistic)
 
+// no accessory
+
+/obj/item/clothing/suit/armor/bulletproof/noacc
+	name = "ballistic vest"
+	desc = "An armored vest with heavy plates to protect against ballistic projectiles."
+	icon = 'icons/clothing/suit/armor/ballistic.dmi'
+	valid_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, ACCESSORY_SLOT_ARMOR_A, ACCESSORY_SLOT_ARMOR_L)
+	restricted_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, ACCESSORY_SLOT_ARMOR_A, ACCESSORY_SLOT_ARMOR_L)
+	body_parts_covered = SLOT_UPPER_BODY|SLOT_LOWER_BODY
+	armor = list(
+		melee = ARMOR_MELEE_RESISTANT,
+		bullet = ARMOR_BALLISTIC_AP,
+		laser = ARMOR_LASER_SMALL,
+		energy = ARMOR_ENERGY_MINOR,
+		bomb = ARMOR_BOMB_PADDED
+		)
+	siemens_coefficient = 0.7
+	material = /decl/material/solid/gemstone/diamond
```

Comment:
Diamond seems like an odd choice, as does titanium, since both are pretty hard to come by. Maybe plasteel instead?

Reply:
True I wanted to keep it from very difficult but I see your point